client/webpack.config.js

```diff
@@ -3,12 +3,15 @@ const path = require('path');
 const webpack = require('webpack');
 const autoprefixer = require('autoprefixer');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const failPlugin = require('webpack-fail-plugin');
 
 
 // Always enabled plugins
 let plugins = [
   // Extract CSS files to the 'bundle.css'.
-  new ExtractTextPlugin('_bundle.css')
+  new ExtractTextPlugin('_bundle.css'),
+  // This plugin should be always required. See https://github.com/webpack/webpack/issues/708
+  failPlugin,
```

Comment:
Plugin order matters but not this case. [`webpack-fail-plugin`](https://github.com/TiddoLangerak/webpack-fail-plugin) will be invoked after all those compilation has been finished.

###### Reference
- https://github.com/TiddoLangerak/webpack-fail-plugin/blob/master/index.js#L9